src/lxc/terminal.c

```diff
@@ -206,7 +206,10 @@ void lxc_terminal_signal_fini(struct lxc_terminal_state *ts)
 	if (ts->sigfd >= 0) {
 		close(ts->sigfd);
 
-		if (sigprocmask(SIG_SETMASK, &ts->oldmask, NULL) < 0)
+		int ret;
```

Comment:
So...  I'm pretty sure this was in our codingstyle, but do we really want to do this?  It's longer and arguably not clearer.

I'll merge it if you really want, but I prefer the old style here.

Reply:
No, I can just remove it.

Reply:
removed :)